docs/user_guide_ova.md

```diff
@@ -0,0 +1,190 @@
+#User Guide of Harbor Virtual Appliance
+##Overview
+This guide walks you through the fundamentals of using Harbor virtual appliance. You'll learn how to use Harbor to:  
+
+* Manage your projects.
+* Manage members of a project.
+* Replicate projects to a remote registry.
+* Search projects and repositories.
+* Manage Harbor system if you are the system administrator:
+ + Manage users.
+ + Manage destinations.
+ + Manage replication policies.
+* Pull and push images using Docker client.
+* Delete repositories and images.
+
+
+##Role Based Access Control
+
+![rbac](img/rbac.png)
+
+Harbor manages images through projects. Users can be added into one project as a member with different privileges. RBAC (Role Based Access Control) is provided in Harbor and there are four roles with different privileges:  
+

```

Comment:
only three roles in a project. 

SysAdmin and anonymous is not project specific, please discuss them separately.